### PR TITLE
theme cleanup

### PR DESCRIPTION
**Pull request by tadjohnston**
No Story

Cleaning up to utilize [react-themed](https://github.com/aribouius/react-themed) properly.

### CODE REVIEW

File: packages/react-ui-core/src/Collapsible/Collapsible.js [hunk 1]
```diff
@@ -56,25 +56,38 @@ export default class Collapsible extends Component {
...
             </div>
-            <div className={theme.alignBottom}>
-              <Button className={cn(theme.strong, theme.btn)} onClick={this.handleClick}>
-                {this.state.display ? visibleText : hiddenText}
-              </Button>
-            </div>
+            <Button
+              onClick={this.handleClick}
+              className={classnames(
+                theme.Collapsible_Button,
```
Comment (lfender6445) [inline packages/react-ui-core/src/Collapsible/Collapsible.js]: I have some filter panel work that will need to be updated post upgrade

**Comment by lfender6445** [comment]
👍  this will need to be shipped with adjustments to rentjs as well so we are using correct classnames.

File: packages/react-ui-map/src/Mapbox/Mapbox.js [hunk 2]
```diff
@@ -104,12 +109,14 @@ export default class Mapbox extends PureComponent {
...
-            className,
-            theme[`Map-${color}`],
-            theme[`Map-${size}`],
-            theme.Map,
-          )}
+          className={
+            classnames(
+              className,
+              theme.Mapbox,
+              color && theme[`Mapbox-${color}`],
```
Comment (ninomichaels) [inline packages/react-ui-map/src/Mapbox/Mapbox.js]: I think the stylesheet needs to be updated as well.

File: packages/react-ui-core/src/Button/Button.js [hunk 3]
```diff
@@ -31,12 +31,14 @@ export default class Button extends Component {
...
       <button
         {...props}
-        className={classNames(
-          className,
-          theme.Button,
-          theme[`Button-${color}`],
-          theme[`Button-${size}`],
-        )}
+        className={
+          classNames(
```
Comment (ninomichaels) [inline packages/react-ui-core/src/Button/Button.js]: did you want to revise this to `classnames` as well?

File: packages/react-ui-core/src/Form/RangeSlider.js [hunk 4]
```diff
@@ -1,10 +1,17 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import InputRange from 'react-input-range'
+import classnames from 'classnames'
+import themed from 'react-themed'
+
+@themed(/^RangeSlider/, {
```
Comment (ninomichaels) [inline packages/react-ui-core/src/Form/RangeSlider.js]: There's a `RangeSlider.css` I think that is going unused or needs to be updated?

File: packages/react-ui-core/src/Button/Button.js (hunk 3 above)
Comment (tadjohnston) [reply packages/react-ui-core/src/Button/Button.js]: Doh! yep...

File: packages/react-ui-core/src/Form/RangeSlider.js (hunk 4 above)
Comment (tadjohnston) [reply packages/react-ui-core/src/Form/RangeSlider.js]: Yeah, I wasn't sure how to handle that file since it's really external.

**Comment by ninomichaels** [comment]
👍

**Pull request merged**